test/src/main/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuilders.java

```diff
@@ -124,6 +137,24 @@ public LogoutRequestBuilder logoutUrl(String logoutUrl, Object... uriVars) {
 
 		private LogoutRequestBuilder() {
 		}
+
```

Comment:
Let's make these public methods above the private constructor to follow Spring Security's conventions